### PR TITLE
Remove libcap/prctl artifacts

### PR DESCRIPTION
**Pull request by ivyl**
They seem like relics of the pasts, from when we were retaining the
ptrace cap.

Some translations still may need updates.

### CODE REVIEW

File: sway/main.c [hunk 1]
```diff
@@ -181,28 +177,8 @@ static void log_kernel() {
...
-			wlr_log(WLR_ERROR,
-				"If unsure, strip the file caps.");
-			exit(EXIT_FAILURE);
-		}
-		free(exe);
-#endif
-}
 
-static void drop_permissions(bool keep_caps) {
+static void drop_permissions() {
```
Comment (ianyfan) [inline sway/main.c]: `drop_permissions(void)`

Comment (ivyl) [reply sway/main.c]: is this a convention here? I neither see other functions in the immediate neighborhood using this, nor CONTRIBUTIN.md mentions it

Comment (emersion) [reply sway/main.c]: In C, `void call_me_maybe()` declares a function that takes an unspecified number of parameters. So a call like `call_me_maybe(1, 2, 3)` is valid.

To declare a function that takes no parameter, one should use `void call_me_maybe(void)`.

Comment (ianyfan) [reply sway/main.c]: @ivyl Leaving it empty is a mistake and should be corrected, along with the other functions in the code.

Comment (ivyl) [reply sway/main.c]: Oh yeah, forgot about that C quirk. Thanks!

2 additional patches to follow this one coming in 5 :-)

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**